### PR TITLE
AC_AttitudeControl: ensure yaw gets a FLTD default

### PR DESCRIPTION
**Pull request by andyp1per**
When people come to tune yaw D they need to have a FLTD set. There is  no downside to not setting this by default

### CODE REVIEW

**Comment by amilcarlucas** [comment]
I have it on the tuning guide, but a default is even better

File: libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.h [hunk 1]
```diff
@@ -137,7 +137,7 @@ class AC_AttitudeControl_Multi : public AC_AttitudeControl {
             .imax      = AC_ATC_MULTI_RATE_YAW_IMAX,
             .filt_T_hz = AC_ATC_MULTI_RATE_RP_FILT_HZ,
             .filt_E_hz = AC_ATC_MULTI_RATE_YAW_FILT_HZ,
-            .filt_D_hz = 0.0f,
+            .filt_D_hz = AC_ATC_MULTI_RATE_RP_FILT_HZ,
```
Comment (IamPete1) [inline libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.h:140]: Its a bit upsetting that its using `RATE_RP` (roll pitch) but filt T is the same....

In fact `filt_E` is useing `AC_ATC_MULTI_RATE_YAW_FILT_HZ` and `filt_T_hz` is using `AC_ATC_MULTI_RATE_RP_FILT_HZ` its all a bit odd....

Comment (lthall) [reply libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.h:140]: Yeh, I can see how this is a little confusing or unclear. What would you suggest?

The end result is we want roll, pitch and yaw filter T and D to be the same (20.0)
roll and pitch filter E to be 0.0
yaw filter E to be 2.0

There is an opportunity here to make this clear for the future.

Comment (andyp1per) [reply libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.h:140]: LOL, I originally did a bigger change that changed the names but figured this was better for 4.5 :-P

**Comment by lthall** [comment]
After the meeting I believe the only change needed for clarity is the name of the variable to include Y for yaw.
AC_ATC_MULTI_RATE_RP_FILT_HZ to AC_ATC_MULTI_RATE_RPY_FILT_HZ

**Comment by peterbarker** [comment]
I've cherry-picked Leonard's fixes for the naming into this PR.

In addition to fixing the defines in AP_CustomControl he's fixed the same defines in Copter.

I've tested that there's no compiler output change vs Andy's patch, so I believe that the one objection brought up at DevCall has been addressed.  I'm marking this as MergeOnCIPass.

**Pull request merged**